content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -195,6 +195,9 @@ metadata:
 spec:
   template:
     # This is the pod template
+    metadata:
+      labels:             
+        id: awesome-label 
```

Comment:
label should be something meaningful, useful ...

Reply:
I have updated the label. Please check.